dom-event-delegation/main.js

```diff
@@ -0,0 +1,14 @@
+const $uL = document.querySelectorAll('ul');
+
+$uL.forEach(function (uL) {
```

Comment:
You shouldn't need to use a `forEach` loop here because there is only one `ul` element on your HTML. You can just use `querySelector` on line 1 and select the `ul` directly.